negotiation/proposed.emu

```diff
@@ -359,13 +359,19 @@
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getbooleanorstringnumberformatoption" aoid="GetBooleanOrStringNumberFormatOption">
-      <h1>GetBooleanOrStringNumberFormatOption ( _options_, _property_, _stringValues_, _fallback_ )</h1>
-
-      <p>
-        The abstract operation GetBooleanOrStringNumberFormatOption extracts the value of the property named _property_ from the provided _options_ object. It returns either *true*, *false*, _fallback_, or one of the elements of the List _values_. The following steps are taken:
-      </p>
-
+    <emu-clause id="sec-getbooleanorstringnumberformatoption" type="abstract operation">
+      <h1>
+        GetOption (
```

Comment:
Wrong operation name here. I'm pretty sure collisions like this are detected by newer versions of ecmarkup.
```suggestion
        GetBooleanOrStringNumberFormatOption (
```